mobile-widgets/qmlprefs.cpp

```diff
@@ -36,13 +35,13 @@ QMLPrefs *QMLPrefs::instance()
 
 qPrefCloudStorage::cloud_status QMLPrefs::credentialStatus() const
 {
-	return m_credentialStatus;
+	return (qPrefCloudStorage::cloud_status)qPrefCloudStorage::cloud_verification_status();
```

Comment:
so the commit message tells me that you are `removing m_cloudCredentials`.
That appears to be incorrect. The thing that you are removing is called `m_credentialStatus`
And what the commit message _should_ tell the reader is actually what the commit is really doing.
Maybe something like `replace m_credentialStatus with accesses to cloud_verification_status in the qPrefCloudStorage class, because... in order to... preparing for...`
I want the commit message to tell me why you are doing what you are doing. And if it talks about what you are doing, it should correctly summarize what you are doing.

Reply:
refrased.